src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -14,16 +14,33 @@ export class DropdownMenu {
   @Prop() icon: boolean = false;
   @Prop() search: boolean = false;
   @Prop() filter: boolean = false;
+  @Event({ bubbles: false }) selectValues: EventEmitter<Object>;
   @State() isShown: boolean = true;
   @Element() el;
 
+
+  handleValues(event) { 
+    this.selectValues.emit(event.detail)
+  }
+
+  toggleCheckbox(target) {
+    target.querySelector('input').checked = !target.querySelector('input').checked
+  }
+
   componentWillRender() {
     const isInsideDropdown = this.el.closest('ifx-dropdown')
     if (isInsideDropdown) {
       this.isShown = false;
     }
   }
 
+  componentDidLoad() { 
+    const dropdownItems = this.el.querySelectorAll('ifx-dropdown-item');
+    for(let i = 0; i < dropdownItems.length; i++) { 
+      dropdownItems[i].addEventListener('ItemValues', this.handleValues.bind(this))
```

Comment:
does it make sense to add a prefix to the event name to not listen to child events "on accident" ?

Reply:
Isn't this exactly why I am turning the bubbling off? But how does adding a prefix (on) stops the event listeners from listening to child-elements?